cfgov/tccp/jinja2/tccp/card.html

```diff
@@ -35,6 +35,25 @@
     <link rel="stylesheet" href="{{ static('apps/tccp/css/main.css') }}">
 {% endblock %}
 
+{% macro data_section(
+    heading,
+    heading_classes="h2 u-mb0",
+    subheading_classes="h4 u-mt0"
+) %}
+<div
+    {%- if heading_classes %} class="{{ heading_classes }}"{% endif -%}
+>
+    {{ caller() }}
+    <span class="u-visually-hidden">{{ heading }}</span>
+</div>
+<div
+    {%- if subheading_classes %} class="{{ subheading_classes }}"{% endif %}
+    aria-hidden="true"
+>
+    {{- heading -}}
```

Comment:
@niqjohnson? This came from your code. Is this for screenreader support?

Reply:
I think that came from @contolini's improvements after an accessibility audit. And looking back at older chats, we'd also talked about changing the markup from `div`s to `span`s to help a screenreader read the text more naturally. @contolini, when you're back, do you want to take a look at this one?

Reply:
Yeah, it's for assistive technologies. TCCP has a novel visual hierarchy that isn't great for screen readers. Each card's data point has its value appear larger and on a new line above its descriptor, e.g. annual fee:

<img width="291" alt="image" src="https://github.com/cfpb/consumerfinance.gov/assets/1060248/17eaffb7-6d89-4454-aa74-b20eda3ae7cf">

A screenreader reads "$99" and then stops because it's a block level element with no relationship to the next line, "Annual fee". I first tried [term and definition roles](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/definition_role) to improve things but the template ended up rather messy and screenreaders don't have great support for those roles. @niqjohnson had the smart suggestion to use `span`s but because there's CSS that makes the `span`s block level, it didn't change the situation. So ultimately the cleanest solution was to add invisible text next to the value so that screenreaders read "$99 Annual fee" when they reach the section.